/**:  # This should match the node name
  ros__parameters:
    gps_data_file: "/home/melike/projects/other_packages_ws/src/data/2024_08_16-route3.txt"

    publishers:
      nav_sat_fix:
        enable: true
        topic: "/ros_pospac_bridge/gps/fix"
      imu:
        enable: true
        topic: "/ros_pospac_bridge/imu_data"
      pose_with_covariance_stamped:
        enable: true
        topic: "/ros_pospac_bridge/pose_with_covariance"
      pose_array:
        enable: false
        topic: "/ros_pospac_bridge/pose_array"
      twist_with_covariance_stamped:
        enable: true
        topic: "/ros_pospac_bridge/twist_with_covariance"
      pose_stamped:
        enable: true
        topic: "/ros_pospac_bridge/pose_stamped"
      tf:
        enable: true

    calibration:
      lidar_to_gnss:
        x: 0.0
        y: 0.0
        z: -0.08
        roll: 0.007237
        pitch: -0.0072323
        yaw: 3.1334945

      base_link_to_lidar:
        x: 0.99
        y: -0.035
        z: 1.31
        roll: 0.0
        pitch: 0.0
        yaw: -3.1095926

      gnss_to_base_link:
        x: -0.998
        y: 0.050
        z: -1.223
        roll: -0.007
        pitch: 0.007
        yaw: -0.024
